stable/grafana/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.podDisruptionBudget }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "grafana.name" . }} 
+  labels:
+    app: {{ template "grafana.name" . }}
+    chart: {{ template "grafana.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.podDisruptionBudget.minAvailble }}
+  minAvailable: {{ .Values.podDisruptionBudget.minAvailble }}
+{{- end }}
+{{- if .Values.podDisruptionBudget.maxUnavailable }}
+  maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
+{{- end }}
+  selector:
+    matchLabels:
```

Comment:
I think the matchLabels should match the matchLabels of the Deployment https://github.com/helm/charts/blob/d4e2858498bc863ba75bffdf482839b4dfc9c8f3/stable/grafana/templates/deployment.yaml#L23-L24

Reply:
I agree, updated.